tests/fetch_and_configure/fetch_and_configure/tasks/iosxr.yml

```diff
@@ -0,0 +1,76 @@
+---
+- debug: msg="START fetch_and_configure/iosxr.yaml on connection={{ ansible_connection }}"
+
+- name: setup - remove interface
+  iosxr_config: &rm
+    commands:
+      - no interface GigabitEthernet0/0/0/2
+  connection: network_cli
+
+- name: setup - clear yang file fetch default path
+  file: &rm_dir
+    path: "~/.ansible/yang"
```

Comment:
Could this be an issues if contributors run the test cases locally, could these lose useful configuration?

https://docs.ansible.com/ansible/latest/modules/tempfile_module.html will give a unique temp directory

Reply:
It shouldn’t be an issue as yang folder is created by the role itself by default in case fetch path is not set by user as role var.

Reply:
It shouldn’t be an issue as yang folder is created by the role itself by default in case fetch path is not set by user as role var.